content/ordering.ts

```diff
@@ -70,7 +70,7 @@ const MODULE_ORDERING: { [key in SectionID]: Chapter[] } = {
   bronze: [
     {
       name: 'Getting Started',
-      items: ['time-comp', 'rect-geo', 'intro-ds', 'simulation'],
+      items: ['time-comp', 'intro-ds', 'simulation', 'rect-geo'],
```

Comment:
thought simulation should be put in front of intro ds? idk lol

Reply:
oh, sure

Reply:
actually uh can you change it back
think ds is more appropriate at the front
really sorry

Reply:
LMAO